src/webhooks/pubsub/slackScores.ts

```diff
@@ -318,6 +367,6 @@ export const triggerSlackScores = async (
     return;
   }
   await sendGitHubEngagementMetrics();
```

Comment:
Unless I'm misunderstanding, you'd want these 3 calls to execute in parallel, rather than one after the other? So maybe:

```suggestion
  await Promise.all[sendGitHubEngagementMetrics(), sendDiscussionMetrics(), sendApisPublishStatus()];
```

Reply:
done